app/models/page.rb

```diff
@@ -4,7 +4,7 @@ class Page < ActiveResource::Base
   self.include_format_in_path = false
   headers["X-API-Token"] = ENV["API_KEY"]
 
-  ANSWER_TYPES = %w[name single_line number address date email national_insurance_number phone_number long_text selection organisation_name text].freeze
+  ANSWER_TYPES = %w[name organisation_name email phone_number national_insurance_number address date selection number text single_line long_text].freeze
```

Comment:
@DavidBiddle do you you think now might be a good time to do the refactor, we spoke about this morning? Splitting this out into simple/complex and then this constant would just be a merge of simple/complex so we have a complete set of answer types?

Reply:
@aliuk2012 I think I'd prefer to do that when we remove the feature flag, as I think there's some related work involved there. But I've renamed `COMPLEX_ANSWER_TYPES` to `ANSWER_TYPES_WITH_SETTINGS` to at least make that clearer here

Reply:
Thank you thats much clearer